### PR TITLE
CIV-9457 LiP Application Confirmation Screen

### PR DESCRIPTION
**Pull request by kenneth-hmcts**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-9457

### Change description ###
Set up LiP Application Confirmation Screen

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/routes/features/generalApplication/submitGeneralApplicationConfirmationController.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import {PageSectionBuilder} from 'common/utils/pageSectionBuilder';
+import {t} from 'i18next';
+import {Claim} from 'models/claim';
+import {convertToPoundsFilter} from 'common/utils/currencyFormat';
+import {getClaimById} from 'modules/utilityService';
+
+const submitGeneralApplicationConfirmationViewPath = 'features/generalApplication/submit-general-application-confirmation';
+const submitGeneralApplicationConfirmationController = Router();
+
+const getGeneralApplicationConfirmationContent = (claimId: string, claim: Claim, lng: string) => {
```
Comment (neeta-hmcts) [inline src/main/routes/features/generalApplication/submitGeneralApplicationConfirmationController.ts:16]: this logic can be moved outside controller

File: src/main/modules/i18n/locales/en.json [hunk 2]
```diff
@@ -2480,6 +2480,16 @@
         "NEED_TO_TELL": "You need to tell us if you agree that the court should make the order requested by the other party. Choose option Yes or No",
         "TEXT_REVIEW": "Make sure you've reviewed the full application. If you disagree with any part of the application, select 'No' and explain why.",
         "TITLE": "Do you agree that the court should make the order that the other parties have requested?"
+      },
+      "CONFIRMATION_PAGE": {
+        "APPLICATION_SAVE": "Your application has been saved, but you need to pay the application fee of £{{ applicationFee }}",
+        "APPLY_HELP_WITH_FEES": "You'll have the option to apply for help with fees once you've clicked 'Pay application fee'.",
+        "CONFIRMATION_TEXT": "You need to pay the application fee to submit the application",
+        "TITLE": "Application created",
```
Comment (manish14836) [inline src/main/modules/i18n/locales/en.json:2488]: Sort these

File: src/main/routes/features/generalApplication/submitGeneralApplicationConfirmationController.ts (hunk 1 above)
Comment (kenneth-hmcts) [reply src/main/routes/features/generalApplication/submitGeneralApplicationConfirmationController.ts]: Updated from review

File: src/main/modules/i18n/locales/en.json (hunk 2 above)
Comment (kenneth-hmcts) [reply src/main/modules/i18n/locales/en.json]: Updated from review

File: src/main/services/features/generalApplication/submitGeneralApplicationConfirmationContent.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import {Claim} from 'models/claim';
+import {convertToPoundsFilter} from 'common/utils/currencyFormat';
+import {DASHBOARD_CLAIMANT_URL, DEFENDANT_SUMMARY_URL} from 'routes/urls';
+import {PageSectionBuilder} from 'common/utils/pageSectionBuilder';
+import {t} from 'i18next';
+
+export const getGeneralApplicationConfirmationContent = (claimId: string, claim: Claim, lng: string) => {
+  const applicationFee = convertToPoundsFilter(claim.generalApplication?.applicationFee?.calculatedAmountInPence?.toString());
+  const dashboardUrl = claim.isClaimant()
+    ? DASHBOARD_CLAIMANT_URL.replace(':id', claimId)
```
Comment (manish14836) [inline src/main/services/features/generalApplication/submitGeneralApplicationConfirmationContent.ts:10]: Use constructResponseUrlWithIdParams method

**Review by neeta-hmcts — APPROVED** [review]

**Review by GorilaNaranja — APPROVED** [review]

**Review by neeta-hmcts — APPROVED** [review]

**Review by manish14836 — DISMISSED** [review]

**Pull request merged**